osu.Game/Screens/Play/Player.cs

```diff
@@ -213,8 +213,8 @@ private void load(AudioManager audio, OsuConfigManager config, OsuGameBase game)
             dependencies.CacheAs(DrawableRuleset);
 
             ScoreProcessor = ruleset.CreateScoreProcessor();
-            ScoreProcessor.ApplyBeatmap(playableBeatmap);
             ScoreProcessor.Mods.Value = gameplayMods;
+            ScoreProcessor.ApplyBeatmap(playableBeatmap);
```

Comment:
It's actually not all that important. I was scared when I first saw the code, which is why I changed it, but it doesn't have any effect.

For example if there was a mod that changed score such as `NoFail`, then setting `Mods` would still recalculate the score immediately. It's not a problem in this case since mods are multiplied at the end rather than into every individual raw value.

Furthermore, this is a thing...:
https://github.com/ppy/osu/blob/538ad1bc98d416d6ce6da3af22a2c59338b8ef3c/osu.Game/Screens/Play/Player.cs#L347-L353

Can add a comment but after knowing this, seems kinda pointless. Either way these two are ordered doesn't make much of a difference.